lib/Parser/Parser.cpp

```diff
@@ -43,6 +43,7 @@ struct Token {
     KnownBits,
     OpenParen,
     CloseParen,
+    RangeOpenParen,
```

Comment:
this isn't a good name for the character that opens a ConstantRange, which is not a paren